lib/project_types/extension/features/argo.rb

```diff
@@ -23,17 +23,23 @@ def create(directory_name, identifier, context)
         Features::ArgoSetup.new(git_template: git_template).call(directory_name, identifier, context)
       end
 
-      def config(context)
+      def config(context, include_renderer_version = true)
         js_system = ShopifyCLI::JsSystem.new(ctx: context)
         if js_system.package_manager == "yarn"
           run_yarn_install(context, js_system)
           run_yarn_run_script(context, js_system)
         end
         filepath = File.join(context.root, SCRIPT_PATH)
         context.abort(context.message("features.argo.missing_file_error")) unless File.exist?(filepath)
+
+        renderer_version = nil
+        if include_renderer_version
+          renderer_version = renderer_package(context).version
```

Comment:
[Doesn't need to be addressed in this PR]

I noticed that we're calling `ShopifyCLI::JsSystem.new(ctx: context)` (once at the beginning of this method, and then again at the beginning of the `renderer_package` method).
Is there a performance hit associated with it? If so, should we change the `renderer_package` method to take `js_system` as an argument (instead of `context`)?